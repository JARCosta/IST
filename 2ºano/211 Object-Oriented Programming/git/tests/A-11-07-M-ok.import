PARTNER|S1|Toshiba|Tokyo, Japan
PARTNER|M1|Rohit Figueiredo|New Delhi, India
BATCH_S|ROLHA|M1|20|500
