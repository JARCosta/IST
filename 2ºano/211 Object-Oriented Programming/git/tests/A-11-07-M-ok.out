Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Parceiros
7 - Gestão de Transacções
8 - Menu de Consultas
9 - Ver Saldo Global
0 - Sair
Escolha uma opção: Gestão de Transacções
1 - Visualizar
2 - Registar desagregação
3 - Registar venda
4 - Registar compra
5 - Receber pagamento de venda
0 - Sair
Escolha uma opção: Identificador do parceiro: Identificador do produto: Preço: Quantidade: Registar receita? Número de componentes: Agravamento: Identificador do produto: Quantidade: Identificador do produto: Quantidade: Registar compra: Operação inválida: O produto 'PAPEL' não existe.
Gestão de Transacções
1 - Visualizar
2 - Registar desagregação
3 - Registar venda
4 - Registar compra
5 - Receber pagamento de venda
0 - Sair
Escolha uma opção: Identificador da transacção: Visualizar: Operação inválida: A transacção '0' não existe.
Gestão de Transacções
1 - Visualizar
2 - Registar desagregação
3 - Registar venda
4 - Registar compra
5 - Receber pagamento de venda
0 - Sair
Escolha uma opção: Opção inválida!
Gestão de Transacções
1 - Visualizar
2 - Registar desagregação
3 - Registar venda
4 - Registar compra
5 - Receber pagamento de venda
0 - Sair
Escolha uma opção: Identificador do parceiro: Identificador do produto: Preço: Quantidade: Registar compra: Operação inválida: O parceiro 'dfdfd' não existe.
Gestão de Transacções
1 - Visualizar
2 - Registar desagregação
3 - Registar venda
4 - Registar compra
5 - Receber pagamento de venda
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Parceiros
7 - Gestão de Transacções
8 - Menu de Consultas
9 - Ver Saldo Global
0 - Sair
Escolha uma opção: Gestão de Produtos
1 - Visualizar todos os produtos
2 - Visualizar todos os lotes
3 - Visualizar lotes fornecidos por parceiro
4 - Visualizar lotes de um dado produto
0 - Sair
Escolha uma opção: ROLHA|20|500
Gestão de Produtos
1 - Visualizar todos os produtos
2 - Visualizar todos os lotes
3 - Visualizar lotes fornecidos por parceiro
4 - Visualizar lotes de um dado produto
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Parceiros
7 - Gestão de Transacções
8 - Menu de Consultas
9 - Ver Saldo Global
0 - Sair
Escolha uma opção: Gestão de Parceiros
1 - Mostrar parceiro
2 - Mostrar parceiros
3 - Registar parceiro
4 - Activa/desactivar notificações de um produto
5 - Mostrar compras com parceiro
6 - Mostrar vendas (e desagregações) com parceiro
0 - Sair
Escolha uma opção: Identificador do parceiro: M1|Rohit Figueiredo|New Delhi, India|NORMAL|0|0|0|0
Gestão de Parceiros
1 - Mostrar parceiro
2 - Mostrar parceiros
3 - Registar parceiro
4 - Activa/desactivar notificações de um produto
5 - Mostrar compras com parceiro
6 - Mostrar vendas (e desagregações) com parceiro
0 - Sair
Escolha uma opção: M1|Rohit Figueiredo|New Delhi, India|NORMAL|0|0|0|0
S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
Gestão de Parceiros
1 - Mostrar parceiro
2 - Mostrar parceiros
3 - Registar parceiro
4 - Activa/desactivar notificações de um produto
5 - Mostrar compras com parceiro
6 - Mostrar vendas (e desagregações) com parceiro
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Parceiros
7 - Gestão de Transacções
8 - Menu de Consultas
9 - Ver Saldo Global
0 - Sair
Escolha uma opção: 